### ISSUE TITLE
Native theme

### ISSUE THREAD

**Issue by lukaszpolowczyk** [zaaack/vscode-markdown-editor#53]
Native theme - e.g. links without underscores, as in VSCode's built-in Markdown Preview.
Adjustment to dark mode.
And so on.

**Comment by wmstack** [comment]
Yup, say for some reason I remapped arrow keys to `hjkl` or `wasd`, for example, under a custom context. One would want those to be the keybindings that are used for the markdown editor as well. However that is not what happens.
